### PR TITLE
README updates for mac users

### PR DESCRIPTION
**Pull request by awadyas**
**NOTE: DO NOT discuss internal CommitChange information in your PR; this PR will be public.
Link back to the issue in the Tix repo when you need to do that.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,12 +75,46 @@ for production but for development, it's likely not a huge issue.
 
 > *Tip*: To run this, add `sudo -u postgres ` to the beginning of the following command.
 
-`createuser houdini_user -s -d -P`
+`createuser admin -s -d -P`
```
Comment (awadyas) [inline README.md:76]: @wwahammy I forget - do we need `houdini_user` AND `admin` roles? Or just `admin` now

File: README.md [hunk 2]
```diff
@@ -73,12 +75,46 @@ for production but for development, it's likely not a huge issue.
...
+nvm use 14
+echo 'export NVM_DIR="$HOME/.nvm"' >> ~/.zshrc
+echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm' >> ~/.zshrc
+echo '[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion' >> ~/.zshrc
+```
 
-Now that we have all of our prerequisites prepared, we need to get the Houdini code.
+Set your Postgres version with homebrew.
 
-`git clone https://github.com/CommitChange/houdini`
```
Comment (awadyas) [inline README.md:80]: I deleted this command here because it's already on line 25

File: README.md (hunk 1 above)
Comment (wwahammy) [reply README.md:76]: Just `admin`.

**Review by wwahammy — APPROVED** [review]
Looks good!

**Pull request merged**